### PR TITLE
Add @akats7 as a Code Owner 

### PR DESCRIPTION
**Pull request by MrAlias**
cc @akats7 

- `go.opentelemetry.io/otel/detectors/aws/ec2`
- `go.opentelemetry.io/otel/detectors/aws/ecs`
- `go.opentelemetry.io/otel/detectors/aws/lambda`
- `go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-lambda-go/otellambda`
- `go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-sdk-go-v2/otelaws`
- `go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux`
- `go.opentelemetry.io/contrib/propagators/aws`

Part of https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5542
Part of https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5543
Part of https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5545
Part of https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5546
Part of https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5547
Part of https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5549
Part of https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5553

### CODE REVIEW

**Review by dmathieu — APPROVED** [review]

**Review by dashpole — APPROVED** [review]

**Comment by MrAlias** [comment]
@akats7 please be sure to approve this after your review it to show you accept the [responsibilities](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/CONTRIBUTING.md#responsibilities) for being a Code Owner for these modules.

**Review by akats7 — APPROVED** [review]

**Review by pellared — APPROVED** [review]

**Review by MadVikingGod — APPROVED** [review]

**Pull request merged**
